packages/app-builder/src/components/hooks/objects/use-object.list.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@apollo/client'
+import { useIntrospection, dummyQuery } from '../use-introspection'
+
+const getOnlyKey = (obj: any) => obj[Object.keys(obj)[0]]
```

Comment:
```suggestion
const getOnlyKey = <T extends any, R extends any>(obj: T): R | undefined => obj[Object.keys(obj)[0]]
```

Is this to get the first entry of the object? `Object.values(obj)[0]` would be a better solution?

Reply:
brain fart, you're right 😅